### PR TITLE
INTMDB-543: LDAP Config and LDAP Verify Resources Fix

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

LDAP Config and LDAP Verify Resources Fix

Link to any related issue(s):

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/provider.go [hunk 1]
```diff
@@ -296,38 +296,87 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
...
+		fmt.Printf("GetSTSRegionalEndpoint error: %s", err)
+		return *config, err
+	}
+
+	defaultResolver := endpoints.DefaultResolver()
+	stsCustResolverFn := func(service, region string, optFns ...func(*endpoints.Options)) (endpoints.ResolvedEndpoint, error) {
+		if service == endpoints.StsServiceID {
+			if endpoint == "" {
+				return endpoints.ResolvedEndpoint{
+					URL:           "https://sts.amazonaws.com",
```
Comment (gssbzn) [inline mongodbatlas/provider.go]: couple questions, does the aws client lib offer this as a const? and if not can we make it a const?

File: mongodbatlas/provider.go [hunk 2]
```diff
@@ -296,38 +296,87 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
...
+	sess := session.Must(session.NewSession(&cfg))
 
 	creds := stscreds.NewCredentials(sess, config.AssumeRole.RoleARN)
 
-	_, _ = sess.Config.Credentials.Get()
-	_, _ = creds.Get()
-	secretString := secretsManagerGetSecretValue(sess, &aws.Config{Credentials: creds, Region: aws.String(region)}, secret)
+	_, err = sess.Config.Credentials.Get()
+	if err != nil {
+		fmt.Printf("Session get credentials error: %s", err)
```
Comment (gssbzn) [inline mongodbatlas/provider.go]: can you please clean up all these prints and make them logs if really needed?

**Review by Zuhairahmed — APPROVED** [review]
LGTM

File: mongodbatlas/provider.go (hunk 2 above)
Comment (martinstibbe) [reply mongodbatlas/provider.go]: set to log.Printf

File: mongodbatlas/provider.go (hunk 1 above)
Comment (martinstibbe) [reply mongodbatlas/provider.go]: set as a const

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**